MathLib/Nonlinear/NewtonRaphson-impl.h

```diff
@@ -67,7 +67,7 @@ bool NewtonRaphson::solve(F_RESIDUAL &f_residual, F_DX &f_dx, const T_VALUE &x0,
             x_new += dx;
             // evaluate residual
             f_residual(x_new, r);
-#ifdef NDEBUG
+#ifdef DEBUG_NR
```

Comment:
The `NR` suffix is too short. `NEWTON_RAPHSON` would be unique and selfexplaining.